src/FSharp.Markdown/Markdown.fs

```diff
@@ -28,19 +30,19 @@ type MarkdownColumnAlignment =
 /// Represents inline formatting inside a paragraph. This can be literal (with text), various
 /// formattings (string, emphasis, etc.), hyperlinks, images, inline maths etc.
 type MarkdownSpan =
-  | Literal of string
-  | InlineCode of string
-  | Strong of MarkdownSpans
-  | Emphasis of MarkdownSpans
-  | AnchorLink of string 
-  | DirectLink of MarkdownSpans * (string * option<string>)
-  | IndirectLink of MarkdownSpans * string * string
-  | DirectImage of string * (string * option<string>)
-  | IndirectImage of string * string * string
-  | HardLineBreak
-  | LatexInlineMath of string
-  | LatexDisplayMath of string
-  | EmbedSpans of MarkdownEmbedSpans
+  | Literal of string * MarkdownRange option
```

Comment:
As this is a large-ish breaking change, it would be great to switch to the F# named parameters on unions, to make it less breaking-change-prone in the future. Could you please add names to the parameters, e.g.:

    | Literal of text:string * range:MarkdownRange option
    | InlineCode of code:string * range:MarkdownRange option
    | Strong of body:MarkdownSpans * range:MarkdownRange option

... etc. I'll be happy to review the naming on the next pass.